FAQ.md

```diff
@@ -0,0 +1,36 @@
+# FAQ
```

Comment:
@dimitrisnl FAQ is still docs, isn't it? Is there some technical reason we prefer FAQ on project root?

Reply:
@dfunckt 	Yes but everything under the `/docs` folder would be a dedicated page under the `[domain]/docs` path. 

The `FAQ.md` as it stands would be parsed and transformed into JSON for use. We wouldn't have a page for it but a small section instead.

You are right though that since it's essentially documentation,so no matter how we want to present it, its place is under `/docs`.

I was following the same pattern as the `landr` projects, but will make sure to revisit it.